examples/social-network/websocket-server/functions/onLikeDislike.js

```diff
@@ -33,6 +33,18 @@ export default async function (event) {
         },
       }
     }],
+    reply: [
```

Comment:
This one shouldn't be changed to `reply` but actually, the other one above (`broadcast`) should be replaced as `send`. Watch out you'll end up with an object with 2 `send` keys so instead, you should combine the two.